ts/features/idpay/initiative/details/store/actions/index.ts

```diff
@@ -38,6 +39,17 @@ export type IdPayTimelineDetailsGetPayloadType = {
   operationId: OperationListDTO["operationId"];
 };
 
+export type IdPayBeneficieryDetailsGetPayloadType = {
```

Comment:
```suggestion
export type IdPayBeneficiaryDetailsGetPayloadType = {
```
small typo, nothing special

Reply:
Fixed in https://github.com/pagopa/io-app/pull/4574/commits/259eea59f144157167d11536f1036760b014ae83